clio-integration-test/src/test/resources/docker-compose/docker-compose-jenkins.yml

```diff
@@ -0,0 +1,14 @@
+version: '2'
+
+services:
+  clio-jenkins-integration-test:
+    image: broadinstitute/clio-integration-test:${CLIO_DOCKER_TAG}
+    environment:
+      ELASTICSEARCH_HTTP_HOST: elasticsearch1.gotc-dev.broadinstitute.org
+    network_mode: "host"
+  clio-server:
+    image: broadinstitute/clio-server:${CLIO_DOCKER_TAG}
+    environment:
+      clio.server.elasticsearch.http-hosts.0.hostname: elasticsearch1.gotc-dev.broadinstitute.org
```

Comment:
This worked for me both locally and on the server, before I change it let's try to figure out why it's working for me, and maybe see if it can work for others too.